### ISSUE TITLE
Fake and scam, it will send the money to dev address 

### ISSUE THREAD

**Issue by 18jad** [PancakeswapFrontRunBot/PancakeswapFrontRunBot#3]
this will send your money to dev address once u deploy and excute it, he did hide his address inside a function that built it when u run it

**Comment by TomiLikesToCode** [comment]
yes it is very sophisticated scam hard to detect fr took 0.0000000000000001 sec to realise its a scam
